templates/dhcpd.conf-extra.erb

```diff
@@ -4,6 +4,8 @@ include "<%= @dhcp_dir %>/dhcpd.hosts";
 include "<%= @dhcp_dir %>/dhcpd.ignoredsubnets";
 <% if @extra_config && !@extra_config.empty? -%>
```

Comment:
Now it's non-optional you can change this to ```<% unless @extra_config.empty? -%>```

Reply:
updated